### PR TITLE
Add static checks to build steps

### PR DESCRIPTION
**Pull request by andrewrota**
Fix issues and run beautify on existing code

### CODE REVIEW

File: src/event/handlers/mouseenter_events.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function contains(container, maybe) {
   // 16 is for a nodeType constant: window.Node.DOCUMENT_POSITION_CONTAINED_BY,
   // but the window.Node object is not available in IE8
   return container.contains ? container.contains(maybe) :
-      !!(container.compareDocumentPosition(maybe) & 16);
+    !!(container.compareDocumentPosition(maybe) & 16);
```
Comment (danieluhl) [inline src/event/handlers/mouseenter_events.js]: Can we make 16 a const rather than relying on a comment to explain?

File: src/utils/logger.js [hunk 2]
```diff
@@ -3,6 +3,7 @@
  *
  * @author    Matt DeGennaro <mdegennaro@wayfair.com>
  */
+/*eslint-env node */
```
Comment (danieluhl) [inline src/utils/logger.js]: Is it standard to have no space after /* ?

**Comment by danieluhl** [comment]
Looks good, mostly formatting updates.

File: src/event/handlers/mouseenter_events.js (hunk 1 above)
Comment (andrewrota) [inline src/event/handlers/mouseenter_events.js]: I'd rather do it in another pr; the only change here is to formatting.

File: src/utils/logger.js (hunk 2 above)
Comment (andrewrota) [inline src/utils/logger.js]: It's just how eslint handles overrides, probably works either way.  See http://eslint.org/docs/user-guide/configuring.html.

File: precompile/tungsten_template/shared_utils.js [hunk 3]
```diff
@@ -73,8 +74,10 @@ module.exports.compileTemplate = function(contents, srcFile) {
       preserveWhitespace: true
     });
   } catch (ex) {
+    /*eslint-disable no-console */
     console.log('Unable to parse ' + (srcFile || contents));
```
Comment (MattDeeg) [inline precompile/tungsten_template/shared_utils.js]: The flag is fine, but maybe we could use the logger module to get around this?
